swagger/oauth.yaml

```diff
@@ -29,6 +29,10 @@ endpoints:
           type: string
           format: password
         - in: formData
+          name: token_type
+          required: false
+          type: string
+        - in: formData
```

Comment:
We are already sending a post body, so it made more send to extend it rather than add a GET parameter. 